doc/interoperability.rst

```diff
@@ -0,0 +1,10 @@
+.. include:: includes/big_toc_css.rst
```

Comment:
I would like to see this be more of a first class citizen. I don't know how to do that well, though :-/

Reply:
Not sure how to do this, help appreciated but will look into it myself too.